README.md

```diff
@@ -581,6 +590,13 @@ ldt1 = 1979-05-27T07:32:00
 ldt2 = 1979-05-27T00:32:00.999999
 ```
 
+Seconds may be omitted if the local instant is at a zero-second mark. In such a
+case, the colon and digits following the minutes is removed.
```

Comment:
Should it be "are removed" rather than "is"? (Likewise below.)

Reply:
Whoops. You're right. Expect a fix in the next few hours.